.github/workflows/build-app.yml

```diff
@@ -4,106 +4,151 @@ on: [push]
 
 jobs:
   build-macos:
-    runs-on: macos-10.15
+    runs-on: macos-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - name: Get yarn cache
-        id: yarn-cache
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
IMHO it's better to use `node-version-file` and reference the existing .nvmrc. That way there's only one source of truth for the node version. In this case, the `.nvmrc` is still set to v12 and should be updated too :). https://github.com/gitify-app/gitify/blob/ci/fix-ci-versions/.nvmrc

Reply:
Was not aware of `node-version-file`, but indeed good change, thanks!